### ISSUE TITLE
actual patchset?

### ISSUE THREAD

**Issue by dada513** [MinecraftMachina/minecraft-arm64-patches#1]
Hey, I may be wrong, but this seems to be only jarfiles, are there any sources for your patches to these libraries? Thanks!

**Comment by ViRb3** [comment]
Hey. Unfortunately, you are not wrong. These libraries change very frequently for experimentation, so I have not committed to providing reproducible builds at the moment. I agree this should be done eventually, though.

**Comment by dada513** [comment]
Thanks. I wanted to take a shot at fixing M1 on [PolyMC](https://github.com/PolyMC/PolyMC). Having these patches to work with would be helpful

**Comment by ViRb3** [comment]
Ah. This was already discussed at https://github.com/PolyMC/PolyMC/issues/21, and I believe we came to the agreement to keep ManyMC as the single M1 fork of MultiMC for now, instead of splitting the community and doubling the developer work?

**Comment by dada513** [comment]
> Ah. This was already discussed at [PolyMC/PolyMC#21](https://github.com/PolyMC/PolyMC/issues/21), and I believe we came to the agreement to keep ManyMC as the single M1 fork of MultiMC for now, instead of splitting the community and doubling the developer work?

Yeah well I will not PR M1 support directly most likely, just trying to fix the core issue of not being able to change LWJGL version properly because now we have some other features which are nice for M1 users too.
BSD systems have this problem too where patched LWJGL is needed so that’s why I’m working on this, also other fixes are included in LWJGL I believe which include keyboard polling.

**Comment by ViRb3** [comment]
I see. For hacking things together on macOS, you can use the libs directly from here :) For Linux, I don't think i can help much unfortunately. A lot of the hacks here are specific to the quirks in macOS. I will try to document the patches that I made eventually, but no promise on how soon that will be, I am pretty busy with other things at the moment.
